feat: add parser for number values

carlovsk opened the pull request:
Adding a simple parser to get number values as `number` type.

This commit will change the number values from this:
```
{
  "a": "1",
  "b": "2",
  "c": "3"
}
```

To this:
```
{
  "a": 1,
  "b": 2,
  "c": 3
}
```

carlovsk committed:
feat: add parser for number values

carlovsk committed:
refactor: rename parameter

jutaz commented on src/index.js:
```diff
@@ -142,5 +145,7 @@ const parseArray = (value) => {
   return noBrackets.split(/,\s*/)
 }
 
+const parseNumber = (value) => Number(value)
```
I'd probably say we should use `parseInt` here:
```suggestion
const parseNumber = (value) => parseInt(value, 10)
```

carlosdnba replied:
That makes sense! Will change it.

carlovsk committed:
fix: use parseInt

this change will turn the parseNumber into an actual parser, instead of a type converter

jutaz reviewed (approved):
Thanks for this fix!

jutaz commented:
The latest (`1.1.0`) has been published: https://www.npmjs.com/package/athena-struct-parser
